migrations/Version202112231326462260_taoQtiTest.php

```diff
@@ -0,0 +1,123 @@
+<?php
+
+declare(strict_types=1);
+
+namespace oat\taoQtiTest\migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use oat\oatbox\event\EventManager;
+use oat\oatbox\reporting\Report;
+use oat\tao\scripts\tools\migrations\AbstractMigration;
+use oat\taoQtiTest\models\classes\event\ResultTestVariablesTransmissionEvent;
+use oat\taoQtiTest\models\classes\$event\ResultTransmissionEventHandler\Api\ResultTransmissionEventHandlerInterface;
+use oat\taoQtiTest\models\classes\$event\ResultTransmissionEventHandler\AsynchronousResultTransmissionEventHandler;
```

Comment:
Something went very wrong here. What's up with that `$event` in the import?

Reply:
unsuccessful mass replace, should be fixed already